lib/send.js

```diff
@@ -546,6 +551,9 @@ function sendFileDirectly (request, path, stat, options) {
 
           // Content-Range
           statusCode = 206
+          if (options.maxChunkSize) {
+            ranges[0].end = Math.min(ranges[0].end, ranges[0].start + options.maxChunkSize - 1)
```

Comment:
It doesnt seem right.

Reply:
There are some unit tests that success, so I think you are pointing out some edge cases that cause bug. If so, could you tell me the condition or something?